### PR TITLE
Press kit page

### PR DESCRIPTION
**Pull request by mbouzi**
- mock - https://www.figma.com/file/g7doUVwCahjA5YHJNQnNd7A5/Tardigrade.io?node-id=1586%3A25
- articles up to date
- correct download links
- page can be located at "/press-kit"

### CODE REVIEW

File: assets/scss/_press-kit.scss [hunk 1]
```diff
@@ -0,0 +1,428 @@
+.press-kit-page {
+  h2 {
```
Comment (VitaliiShpital) [inline assets/scss/_press-kit.scss]: using tags while styling increases rendering time. Please use classes instead

File: layouts/partials/press-kit/colors.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <p>
+              Download Color Kit
+          </p>
+          <img class="img-fluid" src="/img/download.png" alt="benefits icon">
+        </a>
+      </div>
+    </div>
+    <div class="pt-4 row d-flex flex-row">
+      <div class="press-kit__colors-color col-xs-12 col-lg-4">
+        <div class="press-kit__colors-visual" style="background: #2582FF"></div>
```
Comment (VitaliiShpital) [inline layouts/partials/press-kit/colors.html]: using inline styles should be avoided

File: layouts/partials/press-kit/bottom.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{{ with .Site.GetPage "bottom"}}
+<div class="container-fluid">
+  <div class="container">
+    <div class="spacer125"></div>
+    <div class="row">
+      <h2>
+        {{ replace .Params.title "\n" "<br/>" | safeHTML}}
+      </h2>
+      <div class="spacer20"></div>
+      <a class="press-kit__bottom-link" href="#">
```
Comment (VitaliiShpital) [inline layouts/partials/press-kit/bottom.html]: using class nesting could make templates more readable

**Comment by mbouzi** [comment]
@VitaliiShpital all feedback has been addressed!

**Pull request merged**